modules/olmv1-adding-a-catalog.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_content-type: PROCEDURE
+
+[id="olmv1-adding-a-catalog-to-a-cluster_{context}"]
+= Adding a catalog to a cluster
+
+To add a catalog to a cluster, create a catalog custom resource (CR) and apply it to the cluster.
+
+.Procedure
+
+. Create a catalog custom resource (CR), similar to the following example:
++
+.Example `redhat-operators.yaml`
+[source,yaml,subs="attributes+"]
+----
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: redhat-operators
+spec:
+  source:
+    type: image
+    image:
+      ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <1>
+----
+<1> Specify the catalog's image in the `spec.source.image` field.
+
+. Run the following command to add the catalog to your cluster:
++
+[source,terminal]
+----
+$ oc apply -f redhat-operators.yaml
+----
++
+.Example output
+[source,text]
+----
+catalog.catalogd.operatorframework.io/redhat-operators created
+----
+
+.Verification
+
+* Run the following commands to verify the status of your catalog:
+
+.. Run the following command to check if your catalog is available:
++
+[source,terminal]
+----
+$ oc get catalog
+----
++
+.Example output
+[source,text]
+----
+NAME                  AGE
+redhat-operators      20s
+----
+
+.. Run the following command to check the status of your catalog:
++
+[source,terminal]
+----
+$ oc get catalogs.catalogd.operatorframework.io -o yaml
+----
++
+.Example output
+[source,text,subs="attributes+"]
+----
+apiVersion: v1
+items:
+- apiVersion: catalogd.operatorframework.io/v1alpha1
+  kind: Catalog
+  metadata:
+    annotations:
+      kubectl.kubernetes.io/last-applied-configuration: |
+        {"apiVersion":"catalogd.operatorframework.io/v1alpha1","kind":"Catalog","metadata":{"annotations":{},"name":"redhat-operators"},"spec":{"source":{"image":{"ref":"registry.redhat.io/redhat/redhat-operator-index:v4.14"},"type":"image"}}}
+    creationTimestamp: "2023-10-16T13:30:59Z"
+    generation: 1
+    name: redhat-operators
+    resourceVersion: "37304"
+    uid: cf00c68c-4312-4e06-aa8a-299f0bbf496b
+  spec:
+    source:
+      image:
+        ref: registry.redhat.io/redhat/redhat-operator-index:v4.14
```

Comment:
since this block has the attribute sub, did you mean to use `{product-version}` here?
```suggestion
        ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version}
```